sdk/storage/azure-storage-blob/setup.py

```diff
@@ -71,7 +71,7 @@
     author_email='ascl@microsoft.com',
     url='https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/storage/azure-storage-blob',
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```

Comment:
Could you please add the 3.9 classifier to all the releases?
`'Programming Language :: Python :: 3.8'`